api/lib/domain/services/algorithm-methods/flash.js

```diff
@@ -52,35 +53,44 @@ function getEstimatedLevel({ allAnswers, challenges }) {
   }
 
   let latestEstimatedLevel = DEFAULT_ESTIMATED_LEVEL;
-  let samplesProbabilities = Array(samples.length).fill(1);
 
-  for (const answer of allAnswers) {
-    let gaussianValuesForSamples = samples.map((sample) =>
-      _getGaussianValue({ gaussianMean: latestEstimatedLevel, value: sample })
-    );
-    gaussianValuesForSamples = _normalizeDistribution(gaussianValuesForSamples);
+  const samplesWResults = samples.map((sample) => ({
```

Comment:
```suggestion
  const samplesResults = samples.map((sample) => ({
```
Le W c'est une typo ?